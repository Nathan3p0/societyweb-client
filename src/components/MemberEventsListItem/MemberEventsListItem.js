import React from 'react';

const MemberEventsListItem = (props) => {
    return (
        <li className="tableRow" onClick={() => props.findEvent(props.id)}>
            <div className="col col-1" data-label="Event Date">{props.date}</div>
            <div className="col col-2" data-label="Event Name">{props.name}</div>
            <div className="col col-3" data-label="Event Time">{props.time}</div>
            <div className="col col-4" data-label="Customer Phone">{props.location}</div>
        </li>
    );
}

export default MemberEventsListItem;