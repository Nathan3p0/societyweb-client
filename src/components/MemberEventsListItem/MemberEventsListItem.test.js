import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import MemberEventsListItem from './MemberEventsListItem'

describe('Member Events List Item Component Rendering Tests', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<MemberEventsListItem />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders to UI as expected', () => {
        const tree = renderer.create(<MemberEventsListItem />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})