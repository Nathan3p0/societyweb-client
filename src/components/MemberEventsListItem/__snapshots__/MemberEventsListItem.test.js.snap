// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Member Events List Item Component Rendering Tests renders to UI as expected 1`] = `
<li
  className="tableRow"
  onClick={[Function]}
>
  <div
    className="col col-1"
    data-label="Event Date"
  />
  <div
    className="col col-2"
    data-label="Event Name"
  />
  <div
    className="col col-3"
    data-label="Event Time"
  />
  <div
    className="col col-4"
    data-label="Customer Phone"
  />
</li>
`;
